benchmarking/cluster3nodes/benchmark.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/sh
+
+### parameters
+
+# process
+create_new_genesis=1
+clean_explorer_db=1
+run_cluster_nodes=1
+run_explorer=1
+run_tx_generator=1
+
+
+### >>>>>> do not change anything below this point
+
+# check for explorer binary
+if [ ! -d cardano-explorer.git ]; then
+  echo "cardano explorer missing. run the script 'prepare-explorer.sh' first."
+  exit 1
+fi
+
+BASEDIR=$(realpath $(dirname $0))
+
+# clean
+if [ -d db ]; then rm -rf db; fi
+if [ -d logs ]; then rm -rf logs; fi
+if [ -d socket ]; then rm -rf socket; fi
+if [ -d db-0 ]; then rm -rf db-0; fi
+if [ -d db-1 ]; then rm -rf db-1; fi
+if [ -d db-2 ]; then rm -rf db-2; fi
```

Comment:
I'd say that we don't need the tests -- just a single unconditional `rm -rf logs socket db db-{0,1,2}` would do.